src/gsi/intjo.f90

```diff
@@ -240,6 +241,8 @@ subroutine intjo_(rval,qpred,sval,sbias)
 use m_obsdiags, only: obOper_destroy
 use gsi_obOper, only: obOper
 
+use intradmod, only: setrad
```

Comment:
Since `setrad(sval(1))` has been removed the `use intradmod` line is no longer needed.  Please remove this line and the blank line following it.